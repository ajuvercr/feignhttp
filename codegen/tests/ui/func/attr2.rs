use feignhttp_codegen::get;

// error: unknown arg type: bbb
//    |  pub async fn send_get(a: i32, #[bbb] b: i32) -> Result<String> {}
//    |                                  ^^^

#[get("http://xxx")]
pub async fn send_get(a: i32, #[bbb] b: i32) -> Result<String> {}

fn main() {}
