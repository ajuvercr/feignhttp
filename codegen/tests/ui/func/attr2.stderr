error: unknown arg type: bbb
 --> tests/ui/func/attr2.rs:8:33
  |
8 | pub async fn send_get(a: i32, #[bbb] b: i32) -> Result<String> {}
  |                                 ^^^
